### ISSUE TITLE
Wrong constant "Access-Control-Expose-Header" ?

### ISSUE THREAD

**Issue by mhagnumdw** [pac4j/pac4j#1184]
In [HttpConstants.java](https://github.com/pac4j/pac4j/blob/master/pac4j-core/src/main/java/org/pac4j/core/context/HttpConstants.java#L69) there is the constant
```java
String ACCESS_CONTROL_EXPOSE_HEADERS_HEADER = "Access-Control-Expose-Header";
```

The correct one should be `Access-Control-Expose-Headers`, with an 's' at the end, right?

See: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Expose-Headers

**Comment by leleuj** [comment]
Yes, you're right. Would you mind submitting a PR for that? Thanks

**Comment by leleuj** [comment]
Not necessary anymore. Fixed before the 3.2.0 release.
